spec/features/global_announcements_spec.rb

```diff
@@ -35,23 +34,21 @@
       let(:announcements) { build_list(:instance_announcement, 2, instance: instance) }
       before do
         instance.announcements.clear
+        SystemAnnouncement.destroy_all
         announcements.each(&:save!)
         visit root_path
       end
 
       it 'shows the latest announcement' do
-        announcement = announcements.last
-        expect(page).to have_tag('div.global-announcement') do
-          have_tag('div.panel-heading', with_text: announcement.title)
-          have_tag('div.panel-body', with_text: announcement.content)
-        end
```

Comment:
I think our old test was wrong. the block parsed to have_tag is not executed, it will always pass.
Things like this passed in my experiment:
```
        expect(page).to have_tag('div.global-announcement') do
          have_tag('looool,lol', with_text: announcement.title)
          have_tag('div.fasdf', with_text: announcement.content)
        end
```